pkg/apis/v1alpha3/types.go

```diff
@@ -1,12 +1,13 @@
-package v1alpha2
+package v1alpha3
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 // LogRecordType represents the API resource type for Tekton log records.
-const LogRecordType = "results.tekton.dev/v1alpha2.Log"
+const LogRecordType = "results.tekton.dev/v1alpha3.Log"
+const LogRecordTypeV2 = "results.tekton.dev/v1alpha2.Log"
```

Comment:
We need this also because some of the Type stored in DB can be v2.

Reply:
ack @khrm good catch